lib/cryptoexchange/exchanges/local_bitcoins/services/order_book.rb

```diff
@@ -0,0 +1,44 @@
+module Cryptoexchange::Exchanges
+  module LocalBitcoins
+    module Services
+      class OrderBook < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::LocalBitcoins::Market::API_URL}/bitcoincharts/#{market_pair.target.downcase}/orderbook.json"
+        end
+
+        def adapt(output, market_pair)
+          order_book = Cryptoexchange::Models::OrderBook.new
+
+          order_book.base      = market_pair.base
+          order_book.target    = market_pair.target
+          order_book.market    = LocalBitcoins::Market::NAME
+          order_book.asks      = adapt_orders output['asks']
+          order_book.bids      = adapt_orders output['bids']
+          order_book.timestamp = Time.now.to_i
+          order_book.payload   = output
+          order_book
+        end
+
+        def adapt_orders(orders)
+          orders.collect do |order_entry|
+            price, amount = order_entry
+            Cryptoexchange::Models::Order.new(price: price,
+                                              amount: amount,
```

Comment:
@Swingcloud 
https://localbitcoins.com/bitcoincharts/sgd/orderbook.json
https://localbitcoins.com/api-docs/

The 2nd item, which is amount.
It looks like it refers to the maximum trade in the target currency
This is a little hairy and different than exchanges, think we should divide the amount with price.

It is different than usual exchange because someone is allowed to trade less than the orderbook amount

Reply:
sure let me look into this tomorrow

Reply:
@tmlee so we should just record price in this currency, and save that maximum amount somewhere?